annotation-web/src/main/java/eu/europeana/annotation/web/service/impl/BaseAnnotationValidator.java

```diff
@@ -448,7 +455,7 @@ protected void validateTag(Annotation webAnnotation) throws ParamValidationExcep
      */
     protected void validateDescribing(Annotation webAnnotation) throws ParamValidationException {
 	Body body = webAnnotation.getBody();
-
+	if(body==null) return;
```

Comment:
if the argument is null, that is an error case that cannot be ignored. To avoid NPE, a PropertyValidationException must be thrown

Reply:
Improved.